alsongDalsong/ASAudioKit/ASAudioKit/ASAudioPlayer/ASAudioPlayer.swift

```diff
@@ -5,14 +5,17 @@ public enum PlayType {
     case partial(time: Int)
 }
 
-public final class ASAudioPlayer {
+public final class ASAudioPlayer: NSObject, AVAudioPlayerDelegate {
     private var audioPlayer: AVAudioPlayer?
 
-    public init() {}
-
+    public override init() {}
+    
+    public var onPlaybackFinished: (() -> Void)?
+    ///녹음파일을 재생하고 옵션에 따라 재생시간을 설정합니다.
```

Comment:
```suggestion
    /// 녹음파일을 재생하고 옵션에 따라 재생시간을 설정합니다.
```
/// 로 작성하는 주석은 한 칸 띄워주시면 좋아요!
추가로 함수 주석을 작성할 때는 opt+cmd+/ 키를 누르면 자동 주석이 나옵니당